workers/unity/Packages/com.improbable.gdk.core/Utility/EntityBuilder.cs

```diff
@@ -34,6 +41,15 @@ private EntityBuilder()
             entity = new Entity();
         }
 
+        /// <summary>
+        ///     Adds an arbitrary SpatialOS component with data to the entity template.
+        /// </summary>
+        /// <param name="componentData">The SpatialOS component data.</param>
+        /// <param name="writeAccess">The worker attribute which has write access over this component.</param>
```

Comment:
This is hard to say concisely. The attributes doesn't have write access. I think maybe the attribute needed to be granted write access to this component. Eventually can replace it with "the component layer"